packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -40,6 +40,44 @@ import 'theme_data.dart';
 ///
 /// The [clipBehavior] argument must not be null.
 ///
+/// {@tool sample}
+///
+/// This example shows how to use [FlatButton] to create a simple button
+/// in your flutter code.
```

Comment:
also, please remove "in your flutter code", it's redundant. See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose